### PR TITLE
Ttahub 174/update list of topics

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
- Update topic names in the front end javascript constants in which they live
- Write a recurring job to execute a sequelize update function to rename the offending topics
- Added unit test for job

## How to test
Create a new activity report, go to the topics and resources pane

Confirm that the topics are in alphabetical order and that:
- "CLASS: Classroom Organization" 
- "Behavioral/Mental Health / Trauma"
- "Curriculum (Instructional or Parenting)"
- "Environmental Health and Safety / EPRR"
- "Quality Improvement Plan/QIP"

appear.

To test recurring job aside from unit test, you'll need to run the updateTopicNames function and have some of the un-renamed topics in the database. As an example, to run it locally for me, I added it to an API handler so it would run arbitrarily, and ran SQL statements to confirm that the topics had been renamed.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-174

### Every PR

- [x] Meets issue criteria
- [x] JIRA ticket status updated
- [x] Code is meaningfully tested
- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

**Review by AdamAdHocTeam — APPROVED** [review]
Looks good.

File: src/lib/updateTopicNames.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    // within each report, check our array of topics to rename
+    TOPIC_DICTIONARY.forEach((topic) => {
+      // find the index of the old topic, if it exists
+      const index = topics.indexOf(topic.old);
+
+      // -1 if it doesn't exist (Thanks Javascript)
+      if (index !== -1) {
+        // eslint-disable-next-line no-console
+        console.log(`Renaming ${topic.old} to ${topic.renamed} in ${report.id}`);
```
Comment (kryswisnaskas) [inline src/lib/updateTopicNames.js:57]: Any reason not to use` logger.info` instead of `console.log `?

**Comment by kryswisnaskas** [comment]
Looks very good! I verified that the update job worked. Nice! 
Only a minor comment.

**Review by kryswisnaskas — CHANGES_REQUESTED** [review]

**Review by kryswisnaskas — APPROVED** [review]
🚀

**Pull request merged**